docs/contributing.rst

```diff
@@ -321,6 +321,35 @@ Note that if needed, you can skip these checks with:
     git commit --no-verify
 
 
+Auditing accessibility
+======================
+
+The accessibility checking tools are installed along with the rest of the `yarn`
```

Comment:
could you add in a link to `pa11y` and maybe a description of what we mean by "auditing accessibility"? Many devs may have no experience with a11y or UI audits, so this is all new to them. (maybe this is just fixed by linking to the user-facing docs on a11y in this PR?)

Reply:
update on 970ddd576d5711481d402d12b489626f007c93f5